<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>layerDropdown</class>
 <widget class="QDialog" name="layerDropdown">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>291</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>310</width>
    <height>291</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>310</width>
    <height>291</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Filter</string>
  </property>
  <widget class="QPushButton" name="run_button">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>250</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reset_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>D:/Downloads/refresh-ccw.png</normaloff>D:/Downloads/refresh-ccw.png</iconset>
   </property>
  </widget>
  <widget class="QProgressBar" name="progress_bar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>291</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="logo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>291</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>291</width>
     <height>61</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="pixmap">
    <pixmap>D:/Downloads/flter.png</pixmap>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>291</width>
     <height>112</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QComboBox" name="group_dropdown"/>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Select BGY:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="layer_dropdown"/>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Select Geocode:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="geocode_dropdown"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
